static/app/views/performance/transactionSummary/transactionTags/tagsHeatMap.tsx

```diff
@@ -31,14 +52,60 @@ const findRowKey = row => {
   return Object.keys(row).find(key => key.includes('histogram'));
 };
 
+class VirtualReference {
+  boundingRect: DOMRect;
+
+  constructor(element: HTMLElement) {
+    this.boundingRect = element.getBoundingClientRect();
+  }
+  getBoundingClientRect() {
+    return this.boundingRect;
+  }
+
+  get clientWidth() {
+    return this.getBoundingClientRect().width;
+  }
+
+  get clientHeight() {
+    return this.getBoundingClientRect().height;
+  }
+}
+const getPortal = memoize((usesGlobalPortal): HTMLElement => {
```

Comment:
Is this ever called without `useGlobalPortal`? And should these elements be cleaned up afterward?

Reply:
No it shouldn't be now that I think about it. There should only every be one on the page.